Makefile

```diff
@@ -0,0 +1,15 @@
+docker_image = resinos
+
+all: build_docker_image run_test
+
+build_docker_image: Dockerfile
+	@echo '[Info] Building docker image "$(docker_image)"...'
+	docker build -t $(docker_image) .
+
+run_test: env.list
```

Comment:
Does `run_test` require the image to be built? If so, we should add `build_docker_image` as a pre-requisite

Reply:
Can you set this target as the default goal? i.e. https://github.com/resin-io/etcher/blob/master/Makefile#L579

Reply:
Also, this should be called just `test`, which is the convention for Makefile driven projects.